pallets/subtensor/src/lib.rs

```diff
@@ -369,6 +369,10 @@ pub mod pallet {
         0
     }
     #[pallet::type_value]
+    pub fn DefaultNominatorMinRequiredStake<T: Config>() -> u64 {
+        0
```

Comment:
Why is 0 the default? My first assumption is that it should be some significant amount.